src/shortestpaths/bellman-ford.jl

```diff
@@ -37,31 +37,35 @@ function bellman_ford_shortest_paths(
     distmx::AbstractMatrix{T}=weights(graph)
     ) where T<:Real where U<:Integer
 
-    active = Set{U}(sources)
-    sizehint!(active, nv(graph))
+    active = falses(nv(graph))
+    for s in sources
+        active[s] = true
+    end
     dists = fill(typemax(T), nv(graph))
     parents = zeros(U, nv(graph))
     dists[sources] .= 0
     no_changes = false
+    new_active = falses(nv(graph))
 
     for i in one(U):nv(graph)
         no_changes = true
-        new_active = Set{U}()
-        for u in active
+        new_active.chunks .= 0
+        active_vertices = U[v for v in findall(active)]
+        for u in active_vertices
             for v in outneighbors(graph, u)
                 relax_dist = distmx[u, v] + dists[u]
                 if dists[v] > relax_dist
                     dists[v] = relax_dist
                     parents[v] = u
                     no_changes = false
-                    push!(new_active, v)
+                    new_active[v] = true
                 end
             end
         end
         if no_changes
             break
         end
-        active = new_active
+        active.chunks .= new_active.chunks
```

Comment:
this seems kludgy to me. I'm not sure we should be accessing internal fields of a bitarray like this.

Reply:
That is,
```
julia> a = falses(10);

julia> b = a;

julia> a[1] = true
true

julia> b[1]
true
```